### PR TITLE
[12.0][MIG] account_check_printing_report_base

### PR DESCRIPTION
**Pull request by Abranes**
Standard migration to 12.0

Depends on:
* [x] OCA/reporting-engine/pull/278

### CODE REVIEW

**Comment by gustavovalverde** [comment]
Hi @Abranes, as https://github.com/OCA/reporting-engine/pull/278 has been merged. Are you continuing this development here? Or is this ready to be tested?

**Comment by gustavovalverde** [comment]
@pedrobaeza Travis and runbot are complaining on missing **report_wkhtmltopdf_param** which is already merged. How can we rebuild and making both found it as this module is already on v12: https://github.com/OCA/reporting-engine/tree/12.0/report_wkhtmltopdf_param

**Comment by pedrobaeza** [comment]
The author must include in `oca-dependencies.txt` the repository `reporting-engine`

**Comment by Abranes** [comment]
Hi `reporting-engine` added to `oca-dependencies.txt`.

@gustavovalverde Is ready to test.

**Comment by gustavovalverde** [comment]
@pedrobaeza any ideas why the runbot is failing here?

**Comment by pedrobaeza** [comment]
Some warnings:
```
2019-06-13 15:29:19,219 161 WARNING openerp_test odoo.models: The model account.payment.check.report has no _description
2019-06-13 15:29:19,219 161 WARNING openerp_test odoo.models: The model report.account_check_printing_report_base.report_check_base has no _description
```

**Comment by AaronHForgeFlow** [comment]
Hi @Abranes here, (https://github.com/Studio73/account-payment/pull/1) I forwarded port the improvement that remove the dependency of this report from invoices, it is merged in 10.0:https://github.com/OCA/account-payment/pull/261

Would you mind to review it? Thanks

**Comment by AaronHForgeFlow** [comment]
@Abranes sorry for the lint errors, they are fixed here: https://github.com/Studio73/account-payment/pull/2 

BTW, runbot warnings are not related this module

**Comment by fuentes73** [comment]
@aheficent done.
Thanks!

**Comment by AaronHForgeFlow** [comment]
It seems to me the displaying of the check is very different and I am wondering if this was done on purpose:
image

**Comment by fuentes73** [comment]
I guess it might be due BS4 like in this PR https://github.com/OCA/l10n-spain/pull/1065#issuecomment-484404332
@Abranes @rubencr7 can you have a look?

**Comment by andres-pcg** [comment]
Tested. Working.

**Comment by andres-pcg** [comment]
@pedrobaeza i think this one is ready.

**Comment by pedrobaeza** [comment]
I'm not the best to say that, as I haven't reviewed it. Any way, I see runbot red (maybe it was a temporary glitch).

**Comment by max3903** [comment]
@pedrobaeza Runbot is red on Github but it says "Success" at the end:
https://runbot.odoo-community.org/runbot/static/build/3374750-241-40c52e/logs/job_20_test_all.txt

**Comment by pedrobaeza** [comment]
Then it's due to warnings. Please check and fix them.

**Comment by andres-pcg** [comment]
@Abranes seems like there are two warnings stopping this from being merged: 

``2019-06-14 08:37:08,409 161 WARNING openerp_test odoo.models: The model account.payment.check.report has no _description ``

``2019-06-14 08:37:08,409 161 WARNING openerp_test odoo.models: The model report.account_check_printing_report_base.report_check_base has no _description``

**Comment by AaronHForgeFlow** [comment]
HI @andrp92 , does the report shows to you like this https://github.com/OCA/account-payment/pull/241#issuecomment-502039004 to you? I still see it like that.

**Comment by Tardo** [comment]
ping @Abranes
Any update here?

**Comment by gustavovalverde** [comment]
I'm actually having no errors on reports that we had depending on this module; they kept the same styling from v11. On my side; there's nothing wrong to Merge.

**Review by gustavovalverde — APPROVED** [review]

File: account_check_printing_report_base/models/account_payment_check_report.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# (http://www.eficent.com)
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AccountPaymentCheckReport(models.Model):
+    _name = "account.payment.check.report"
+
```
Comment (MiquelRForgeFlow) [inline account_check_printing_report_base/models/account_payment_check_report.py:11]:
```suggestion
    _description = 'Account Payment Check Report'

```

File: account_check_printing_report_base/report/check_print.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import time
+from odoo import api, models
+from odoo.tools import float_is_zero
+
+
+class ReportCheckPrint(models.AbstractModel):
+    _name = 'report.account_check_printing_report_base.report_check_base'
+
```
Comment (MiquelRForgeFlow) [inline account_check_printing_report_base/report/check_print.py:14]:
```suggestion
    _description= 'Report Check Print'

```

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

**Comment by JordiBForgeFlow** [comment]
Superseeded by #277

**Pull request closed without merging**